src/content/content-safari.js

```diff
@@ -4,19 +4,25 @@ import StateMessage from '@/lib/messaging/message/state-message'
 import MessagingService from '@/lib/messaging/service-safari.js'
 import { TabScript, UIController, LocalStorageArea, HTMLPage } from 'alpheios-components'
 import ComponentStyles from '../../node_modules/alpheios-components/dist/style/style.min.css' // eslint-disable-line
+import Package from '../../package.json'
 
 let uiController = null
 
 /**
  * State request processing function.
  */
 let handleStateRequest = function handleStateRequest (message) {
-  // let browserManifest = browser.runtime.getManifest() // TODO: Do we need this in Safari?
   if (!uiController) {
     let state = new TabScript()
     state.status = TabScript.statuses.script.PENDING
     state.panelStatus = TabScript.statuses.panel.CLOSED
-    uiController = new UIController(state, LocalStorageArea, {}/*, browserManifest */)
+    // TODO: Ignore build number for now
```

Comment:
@kirlat the build number is important for Monica's QA process. Could we just add this as a variable we updated manually in the package.json for now?

Reply:
Yes, definitely

Reply:
Updated with 4b47a5cf21a3640f82e65c40fcd42df4744f995a